src/components/shared/TheButton.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <button
+    :class="{
+      'flex items-center grow justify-center gap-2 py-[10px] px-4 rounded-3xl':
+        regularButton,
+      'text-sm': isInForm
```

Comment:
Which one?

Because, if you mean the `text-sm` one, I'm afraid prettier doesn't agree with you. Unless you want me to disable prettier for this line?